hal/src/boron/network/network.cpp

```diff
@@ -125,7 +125,7 @@ int if_init_platform(void*) {
 #if PLATFORM_ID == PLATFORM_BORON
         en2 = new WizNetif(HAL_SPI_INTERFACE1, D5, D3, D4, mac);
 #elif PLATFORM_ID == PLATFORM_ESOMX
-        en2 = new WizNetif(HAL_SPI_INTERFACE1, D5, D3, D4, mac);        
+        en2 = new WizNetif(HAL_SPI_INTERFACE1, SS1, MOSI1, MISO1, mac);        
```

Comment:
I'd propose we don't use pins that overlap with a peripheral, it is that way for other Gen 3 platforms, but shouldn't be the case here if avoidable.

Reply:
Do we need to test this functionality with the new pins?

Reply:
Ideally yes, but since there is no ethernet wing for the esomx it might take some extra effort to wire up. I'm not even sure this feature will be used for this platform.